pkg/cluster/api/pdapi.go

```diff
@@ -614,23 +617,24 @@ func (pc *PDClient) DelStore(host string, retryOpt *utils.RetryOption) error {
 		}
 	}
 	if err := utils.Retry(func() error {
-		currStores, err := pc.GetStores()
+		currStore, err := pc.GetCurrentStore(host)
```

Comment:
The err will not be nil if we don't found target store, and the Retry will retry the action and fail again